easybuild/easyconfigs/m/medaka/medaka-1.4.3-TF-version-fix.patch

```diff
@@ -0,0 +1,15 @@
+Removing hard-coded version for TensorFlow
```

Comment:
It's easier to deal with this with a `sed` command in `preinstallopts` for the TensorFlow extension.

Also, we should refer to https://github.com/nanoporetech/medaka/issues/323